homeassistant/components/zwave_js/climate.py

```diff
@@ -56,28 +58,6 @@
     ThermostatMode.DRY,
 ]
 
-THERMOSTAT_MODE_SETPOINT_MAP: dict[int, list[ThermostatSetpointType]] = {
-    ThermostatMode.OFF: [],
-    ThermostatMode.HEAT: [ThermostatSetpointType.HEATING],
-    ThermostatMode.COOL: [ThermostatSetpointType.COOLING],
-    ThermostatMode.AUTO: [
-        ThermostatSetpointType.HEATING,
-        ThermostatSetpointType.COOLING,
-    ],
-    ThermostatMode.AUXILIARY: [ThermostatSetpointType.HEATING],
-    ThermostatMode.FURNACE: [ThermostatSetpointType.FURNACE],
-    ThermostatMode.DRY: [ThermostatSetpointType.DRY_AIR],
-    ThermostatMode.MOIST: [ThermostatSetpointType.MOIST_AIR],
-    ThermostatMode.AUTO_CHANGE_OVER: [ThermostatSetpointType.AUTO_CHANGEOVER],
-    ThermostatMode.HEATING_ECON: [ThermostatSetpointType.ENERGY_SAVE_HEATING],
-    ThermostatMode.COOLING_ECON: [ThermostatSetpointType.ENERGY_SAVE_COOLING],
-    ThermostatMode.AWAY: [
-        ThermostatSetpointType.AWAY_HEATING,
-        ThermostatSetpointType.AWAY_COOLING,
-    ],
-    ThermostatMode.FULL_POWER: [ThermostatSetpointType.FULL_POWER],
-}
-
```

Comment:
This seems a not related change

Reply:
I suggest you open a separate PR for this

Reply:
Agreed - I've reverted this change and raise a separate PR. Thanks!

Reply:
PR: #96058